packages/infra/lib/ihe-stack.ts

```diff
@@ -59,21 +59,18 @@ export class IHEStack extends Stack {
 
     // TODO 1377 Setup WAF
 
-    // get the certificate form ACM
-    const certificate = cert.Certificate.fromCertificateArn(
-      this,
-      "IHECertificate",
-      props.config.iheGateway.certArn
-    );
-
     // add domain cert + record
-    const iheApiUrl = `${props.config.iheGateway?.subdomain}.${props.config.domain}`;
-    api.addDomainName("IHEAPIDomain", {
+    const iheApiUrl = `${props.config.iheGateway.subdomain}.${props.config.domain}`;
+    const certificate = new cert.Certificate(this, `IHECertificate`, {
```

Comment:
why is this changing?

AFAIK we had this here on purpose so that we had the sequoia certs loaded on the IHE GW endpoint - this is required so that implementers are able to talk to our gateway

@jonahkaye will have more details, but we'll probably need to keep this as-is

Reply:
Yah so the change here would be creating an AWS issued public certificate, but the way we had it before was referencing our private Sequoia issued certificate. I think we need to keep what we had before so client's can validate they trust us

Reply:
Yeah, I missed the fact we can't import a cert through CDK. I'm reverting this.